qa/L0_perf_resnet/run_test.sh

```diff
@@ -67,7 +67,7 @@ fi
 
 set +e
 
-PERF_CLIENT_PROTOCOL_ARGS="-i grpc -u localhost:8001" &&
+PERF_CLIENT_PROTOCOL_ARGS="-i grpc" &&
```

Comment:
I meant you can drop PERF_CLIENT_PROTOCOL_ARGS altogether and invoke perf_client like this:

`$PERF_CLIENT -v -i ${PERF_CLIENT_PROTOCOL} -m $MODEL_NAME -p5000 -b${STATIC_BATCH}`

You can make the same change in L0_perf_nomodel to maintain the homogeneity.